docs/paper/dev/misc/internal-code.mdx

```diff
@@ -46,6 +46,13 @@ of Paper. This is unlikely to be the case in most production environments. This
 include the version. For example, `org.bukkit.craftbukkit.v1_20_R2.CraftServer` is the full class and package name
 for the CraftServer class in version 1.20.2. You can access these classes easily with some reflection utilities.
 
+:::warning[Removal of relocation in 1.20.5]
+
+As of 1.20.5, Paper removed the relocation of the CraftBukkit package into a versioned package.
+This means that CraftBukkit packages will now always be located in `org.bukkit.craftbukkit` and no longer in `org.bukkit.craftbukkit.<version>`.
```

Comment:
```suggestion
CraftBukkit packages are now located in `org.bukkit.craftbukkit` and not in `org.bukkit.craftbukkit.<version>`.
```

Reply:
addressed your requests. I just added an "and" to connect both sentences.